inc/classes/class-imagify-admin-ajax-post.php

```diff
@@ -202,20 +202,20 @@ protected function optimize_missing_sizes( $media_id, $context ) {
 	 * @param  string $context  The context.
 	 * @return bool|WP_Error    True if successfully launched. A \WP_Error instance on failure.
 	 */
-	protected function generate_webp_versions( $media_id, $context ) {
+	protected function generate_nextgen_versions( $media_id, $context ) {
```

Comment:
:warning: this function has already been refactored here : 
https://github.com/wp-media/imagify-plugin/blob/ffcbe66cfa06498264a8897d398357a7a9b97d7d/inc/classes/class-imagify-admin-ajax-post.php#L205